test/smoke/harness/harness.go

```diff
@@ -314,7 +314,9 @@ func (h *Harness) Start() {
 	h.TwilioNumber("") // register default number
 	h.slack.SetActionURL(h.slackApp.ClientID, h.backend.URL()+"/api/v2/slack/message-action")
 
-	go h.backend.Run(context.Background())
+	go func(h *Harness) {
+		require.NoError(h.t, h.backend.Run(context.Background())) // can't pass to test fatal on error
```

Comment:
We can't use `require` here because it calls `FailNow` which can't be called from a separate goroutine, use `assert`.

https://pkg.go.dev/testing#T.FailNow

https://medium.com/freethreads/golang-testify-require-vs-assert-b3bbfb4e0b8f

Reply:
done.  I just used require to be consistent with the rest of the method.